doc/DataPlaneCodeGeneration/Subclients.md

```diff
@@ -0,0 +1,356 @@
+# Customizing the generated code
+
+## Subclients
+
+autorest requires each operation in swagger file to define a unique string parameter `operationId` (this is different from [official OpenAPI specification](https://swagger.io/docs/specification/paths-and-operations/#operationId)). If parameter value contains underscore, then part after underscore will be treated as operation name, and part before underscore will be treated as the name of the operation group to which this operation belongs. Otherwise, operation will be attributed to the group without a name::
+
+```js
+"paths": {
+  "/namedgroup/op": {
+    "get": {
+      "operationId": "NamedGroup_Op1",
+    }
+  },
+  "/nogroup/op": {
+    "get": {
+      "operationId": "Op2",
+    }
+  }
+}
+```
+
+```yaml
```

Comment:
Is this modeler4 code?  It could be difficult for people who do not know M4 to understand this part.

Reply:
I suggest we could remove that. WIthout this, i could still understand the content.